packages/test-helpers/test/mainnet/faucet.test.ts

```diff
@@ -19,11 +18,11 @@ describe('mainnet: Test faucet claim', function () {
   snapshotAndRevertEach();
 
   const testCases: { tokenOrAddress: common.TokenOrAddress; amount: string; faucet?: string }[] = [
-    { tokenOrAddress: mainnetTokens.ETH, amount: '1' },
-    { tokenOrAddress: mainnetTokens.WETH, amount: '1' },
+    { tokenOrAddress: common.mainnetTokens.ETH, amount: '1' },
+    { tokenOrAddress: common.mainnetTokens.WETH, amount: '1' },
     { tokenOrAddress: '0xBe9895146f7AF43049ca1c1AE358B0541Ea49704', amount: '1' }, // cbETH
     { tokenOrAddress: '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0', amount: '1' }, // wstETH
```

Comment:
```suggestion
    { tokenOrAddress: mainnetTokens.cbETH, amount: '1' },
    { tokenOrAddress: mainnetTokens.wstETH, amount: '1' },
```

Reply:
I think this is intended to test address as function input lol.

Reply:
oh..you are right XD